File: csharp/DVSClient/DVSClient/server/address/suggestions/RestApiSuggestionsFormatResult.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Suggestions
{
    public class RestApiSuggestionsFormatResult
    {
        [JsonProperty("more_results_available")]
        public bool MoreResultsAvailable { get; set; }

        [JsonProperty("confidence")]
        public string? Confidence { get; set; }

        [JsonProperty("suggestions")]
        public IEnumerable<RestApiSuggestionsFormatSuggestion>? Suggestions { get; set; }
    }
}

File: typescript/src/server/address/suggestions/restApiSuggestionsFormatResult.ts
import { RestApiSuggestionsFormatSuggestion } from "./restApiSuggestionsFormatSuggestion";

export type RestApiSuggestionsFormatResult = {
    more_results_available: boolean;
    confidence?: string;
    suggestions?: RestApiSuggestionsFormatSuggestion[];
};